youtube_dl/extractor/twitch.py

```diff
@@ -216,6 +219,14 @@ def _real_extract(self, url):
             item_id, 'mp4')
         self._prefer_source(formats)
         info['formats'] = formats
+
+        info['start_time'] = None
+        parsed_url = compat_urllib_parse.urlparse(url)
+        for component in [parsed_url.fragment, parsed_url.query]:
```

Comment:
No idea, I've never used the official twitch client. Just copied the code from the youtube.py script and figured it would probably the same-ish.